spec/jobs/distribution_reminder_mailer_job_spec.rb

```diff
@@ -21,4 +21,48 @@
       end
     end
   end
+
+  describe "conditionally sending the emails" do
+    let(:organization) { create :organization }
+
+    let(:past_distribution) { create(:distribution, issued_at: Time.zone.now - 1.week) }
+    let(:future_distribution) { create(:distribution, issued_at: Time.zone.now + 1.week) }
+
+    let(:partner_with_no_reminders) { create :partner, send_reminders: false }
+    let(:partner_with_reminders) { create :partner, send_reminders: true }
+    let(:distribution_without_reminder) { create(:distribution, partner: partner_with_no_reminders) }
+    let(:distribution_with_reminder) { create(:distribution, partner: partner_with_reminders) }
+
+    it "does not send mail for past distributions" do
+      Sidekiq::Testing.inline! do
+        expect do
+          DistributionReminderJob.perform_async(past_distribution.id)
+        end.to_not change { ActionMailer::Base.deliveries.count }
+      end
+    end
+
+    it "sends mail for future distributions" do
+      Sidekiq::Testing.inline! do
```

Comment:
Apparently this call only works within it blocks, and I have to make it each time??